### PR TITLE
Add dominatrix borg module to lustwish

### PR DESCRIPTION
**Pull request by thegrb93**
## About The Pull Request
Adds a dominatrix borg module to lustwish (unfortunately someone still has to purchase it for the borg)
The module, when installed, adds a kinky shocker, whip, paddle, and tickle feather to the borg's repertoire

## Why It's Good For The Game
Gives borgs some kinky items for erp

## Proof Of Testing
image
image

## Changelog

:cl:
add: Borg dominatrix module to lustwish that gives them some dominatrix toys
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
Encourages borgs to drop their jobs at best and encourages them to use fetish gear in combat at worst.

👎🏻

**Comment by YakumoChen** [comment]
Another great module for borgs to pref break me with!

**Comment by ReturnToZender** [comment]
Pretty sure these modules respect your preferences by default. If not, be sure they do.

Cool and all as an upgrade set but I wish it was a whole module with sprites. Would be cool as heck to see latex borgs.

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Borgs already suffer from hilarious item bloat, please make this into one radial menu item if possible

**Comment by thegrb93** [comment]
> Borgs already suffer from hilarious item bloat, please make this into one radial menu item if possible

That would make using multiple toys a pain. Only borgs that want the module will have it so no harm done

**Comment by thegrb93** [comment]
Maybe I can do that, but I'm adding more toys if I do.

**Comment by thegrb93** [comment]
@Majkl-J I don't think this is possible actually, requires a bunch of work on TG's side to do. Nova merged it btw https://github.com/NovaSector/NovaSector/pull/359

**Comment by YakumoChen** [comment]
Just because they instant-merged a PR and have no standards doesn't mean we can't have some

**Comment by thegrb93** [comment]
> Just because they instant-merged a PR and have no standards doesn't mean we can't have some

I don't know what you have against borgs but its pretty silly. Both your complaints apply to non-borg characters too y'know?


> Pretty sure these modules respect your preferences by default. If not, be sure they do.
> 
> Cool and all as an upgrade set but I wish it was a whole module with sprites. Would be cool as heck to see latex borgs.

Eventually :>, and yes the items contain the ERP interaction checks.

**Comment by StrangeWeirdKitten** [comment]
The items respect your prefs, so it's not entirely an awful feature. But: 

- There's 5 borg slots open and posi-brains. There's plenty of borgs to go around, I'm not worried about their jobs
- People are not turning on and off their ERP pref. So yes, I can see a borg using these things in combat.
- Our playerbase loves to abuse any inch you give them. Nova and our server are very diffrent. 

If you can make these items a subtype, and remove the actual knockdowns / stamina, then there will be no further concerns.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
I had something written here, but I thought of something else. Ignore this, it's under discussion.

**Comment by StrangeWeirdKitten** [comment]
Pref-breaking is not an issue with this PR, and I will be marking any posts that bring it up as off-topic.

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm
Comment (StrangeWeirdKitten) [inline modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: This entire things needs to either go on Skyrat/skyrat PR (which is our upstream) or be modularized. We do not stay up to date with Nova.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/borgs/code/robot_upgrade.dm
Comment (thegrb93) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: The upstream needs to be updated, the community is playing on code from https://github.com/NovaSector/NovaSector now

Comment (StrangeWeirdKitten) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: I'm telling you what our upstream currently is for the purposes of your PR. Either take it to skyrat/skyrat or modularize it.

Comment (thegrb93) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: I can open a PR there, but will be surprised if there any devs to look at it.

Comment (theselfish) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: Huh. I thought we were disconnecting from Skyrat.

Comment (thegrb93) [reply modular_skyrat/modules/borgs/code/robot_upgrade.dm:1]: @StrangeWeirdKitten https://github.com/Skyrat-SS13/Skyrat-tg/pull/26100

**Comment by StrangeWeirdKitten** [comment]
Merged upstream.

**Pull request merged**